lib/ansible/modules/cloud/hcloud/hcloud_floating_ip_info.py

```diff
@@ -61,6 +61,11 @@
             returned: always
             type: int
             sample: 1937415
+        name:
+            description: Name of the Floating IP
+            returned: Always
+            type: str
+            sample: my-floating-ip
```

Comment:
please append the version_added field:

~~~diff
+           version_added: "2.10"